packages/reporter/src/main.tsx

```diff
@@ -138,10 +138,12 @@ declare global {
   }
 }
 
+const _window = window as any
+
 // NOTE: this is for testing Cypress-in-Cypress
-if (window.Cypress) {
-  window.state = appState
-  window.render = (props) => {
+if (_window.Cypress) {
```

Comment:
Could you add a comment about why the `_` ?
Is it to avoid TypeScript validation?

If it is, use expect error more than `any` if you can.
You could also augment the `window` object if you need to.

Reply:
This was before I added `lib: ["DOM"]` to the `tsconfig.json`, I can remove the these changes